features/enqueuing_products.feature

```diff
@@ -40,7 +40,6 @@ Feature: Enqueuing products
     When I browse products
     And I schedule an Akeneo PIM import for the "Braided hat m" product
     Then I should be notified that it has been successfully enqueued
-    And I should see 1, not imported, item in the Akeneo queue items list
```

Comment:
Why you removed these steps? You cannot check if there is exactly X entry in the symfony_messenger queue/table?

Reply:
The Messenger context cannot be used to verify that messages have been queued. I think it is because, after enqueueing messages, the controller makes a redirect that evidently "clears" the messages contained in the in-memory transport.
The only way to verify that they are queued is by checking the database. Still, at that point, you should make an assertion on the DB directly (even the MessengerContext is a "code assertion"), but above all, I should make the assertions on the DB also for the command. Otherwise, I would have to configure two different messengers for controller and command.
@webgriffe/wg-devs what do you say?

Reply:
@webgriffe/wg-devs guys?

Reply:
The fact that there is a queue should be completely transparent to the user. So I agree with the removal of this step.

Reply:
I agree with @mmenozzi: no need to check the number of items in queue.

Reply:
Ok, i expressed myself badly int he first comment. I didn't mean that we should test if the item to import is in a queue (or any other implementation), but that we MUST assert that this line is called

https://github.com/webgriffe/SyliusAkeneoPlugin/pull/134/files#diff-c9a07a6fedea9e5acbb54d17d14ba8622ddb2a7dfc4f3ebbf4058ef0e00a3a96R126

> $this->messageBus->dispatch($itemImport);

As of now, if i remove this line from the code the build will still pass. This is bad.
Maybe we could stub the MessengerBus and assert that the dispatch was called with a proper object as argument.

Reply:
In a Behat test we should assert only something that the real user would be able to verify himself.

A real used here would run the consume command and would check if the product is imported or not.

We can do the same here, maybe configuring the messenger in synchronous mode for the test environment (which is the default if no transport is configured).